core/ViewDataTable/Config.php

```diff
@@ -463,6 +463,14 @@ class Config
     public $export_limit = '';
 
     /**
+     * Message to show if not data is available for the report
+     * Defaults to `CoreHome_ThereIsNoDataForThisReport` if not set
+     *
+     * @var string
+     */
```

Comment:
A comment would be very good / needed that this message will be printed raw and that user needs to escape output